UnitTests/AABBTest.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include "UnitTests.h"
+#include "../GeometryCore/include/Point.h"
+#include "../GeometryCore/include/AABB.h"
+#include "../GeometryCore/include/Polygon.h"
+
+using namespace geom_utils;
+
+DEFINE_TEST_CASE(Calculate) {
+
+        Polygon poly = { FPoint2D(1, 2), FPoint2D(3, 4), FPoint2D(5, 6) };
+        AABB2D aabb2d(poly);
+        FPoint2D expectedMaxAnswer(5, 6);
+        FPoint2D expectedMinAnswer(1, 2);
+        TEST_ASSERT((aabb2d.min() == expectedMinAnswer && aabb2d.max() == expectedMaxAnswer), "Failure");
+      
+       
+}
+DEFINE_TEST_CASE(isNotValid) {
+  
+    Polygon poly;
+    AABB2D aabb2d(poly);
+    TEST_ASSERT(aabb2d.isValid(aabb2d), "AABB must be not valid!");
```

Comment:
So, `poly` is empty, i.e. it contains no any points... why its bounding box should be valid? Lets clarified this. **Valid bounding box** means it describes bounds for some real non-empty object in 2D space, in other case it is not valid.